src/pages/trails/ballots/view/BallotView.vue

```diff
@@ -384,25 +384,24 @@ export default {
                                                     div(v-if="getPartOfTotal(option)") {{ getPartOfTotalPercent(option) }}%&nbsp
                                     div.linear-progress(v-if="displayWinner(ballot)")
                                         q-linear-progress(rounded size="6px" :value="getPartOfTotal(option)" color="$primary")
-                            q-item(v-if="ballot.status !== 'cancelled' && isBallotOpened(ballot)").capitalize.options-btn
-                                q-item-section.btn-wrapper
-                                    btn(
+                            q-item(v-if="ballot.status !== 'cancelled' && isBallotOpened(ballot)").options-btn
+                                q-btn(
+                                    no-caps
+                                    :color="userCanVote ? 'primary' : 'info'"
```

Comment:
I would not change the color of the button, but manipulate its `opacity` to indicate a disabled button. Perhaps the default `:disable` behavior from Quasar would be enough 🤔

Reply:
When you first open the dialog the 'vote' button is disabled by default until you select an option.
The disabled state for primary color is still a very strong color and can be misunderstood as a not working button instead of a disabled one. That's why I decided to also change the color to make more emphasis on the disabled state.

![image](https://user-images.githubusercontent.com/4420760/194332597-7f63a082-ba62-414f-8a09-d25b42af365b.png)

Please, tell me if you still prefer to use only the primary color in both `disabled` and `normal` states. I will wait for that confirmation to revert this change.

Reply:
oh yeah, I think you're right! Let's leave the gray one for now.